velox/core/PlanNode.h

```diff
@@ -659,12 +659,16 @@ using PartitionFunctionFactory =
 /// different from input.
 class LocalPartitionNode : public PlanNode {
  public:
+  enum class Type { kGather, kRepartition, kReplicate };
```

Comment:
Are you planning to add support for `kReplicate` in this PR?

Reply:
I don't plan to support kReplicate just yet.

Reply:
We should remove this or throw NYI if `kReplicate` is specified?